revolv/base/models.py

```diff
@@ -21,7 +22,9 @@ def create_user_as_admin(self, *args, **kwargs):
         return profile
 
     def get_subscribed_to_newsletter(self, queryset=None):
```

Comment:
Oh looks like this does return a `QuerySet`. Would be helpful to have a `:return:` saying that it does also